WTWSMS/events/ACR_celtic_flavor_events.txt

```diff
@@ -409,6 +436,7 @@ character_event = {
 		}
 		hidden_tooltip = {
 			character_event = { id = ancrel.0210 days = 10 }
+			remove_trait = traveling
```

Comment:
Yes. Travelling is a speficic mechanic for characters _that travel_. The character is traveling.